### PR TITLE
feature: Move gitlabEnterprise configuration to file CY-1354

### PR DESCRIPTION
**Pull request by jllopes**

### CODE REVIEW

File: docs/configuration/git-providers/gitlab-enterprise.md [hunk 1]
```diff
@@ -1,24 +1,34 @@
...
-Set your configuration values for your GitLab instance on the `values.yaml` file.
-**Please note that you must go to `http://codacy.example.com/admin/integrations`, select the desired provider and `Test & Save` your configuration for it to be applied.**
-
 ## GitLab Application
 
-To integrate with GitLab we use a GitLab Application.
+Follow the instructions below to set up the Codacy integration with GitLab Cloud:
 
-Create an application pointing to your local Codacy deployment URL.
+1.  Create a new application in `<gitlabEnterprise url>/profile/applications`, where `<gitlabEnterprise url>` is the url of your own GitLab Enterprise instance, pointing to your local Codacy deployment URL with `api`, `read user` and `read repository` scopes.
```
Comment (pauloribeiro-codacy) [inline docs/configuration/git-providers/gitlab-enterprise.md:7]:
```suggestion
1.  Create a new application in `<gitlabEnterprise url>/profile/applications`, where `<gitlabEnterprise url>` is the URL of your own GitLab Enterprise instance, pointing to your local Codacy instance URL with `api`, `read user` and `read repository` scopes.
```

File: docs/configuration/git-providers/gitlab-enterprise.md [hunk 2]
```diff
@@ -1,24 +1,34 @@
...
 
-You'll need to add the following 'Redirect URI'. Make sure to update your protocol to use either http or https and your domain name as well. Keep in mind this field is case sensitive.
+    You'll need to add the following 'Redirect URI'. Make sure to update your protocol to use either http or https and your hostname as well. Keep in mind this field is case sensitive.
```
Comment (pauloribeiro-codacy) [inline docs/configuration/git-providers/gitlab-enterprise.md:9]:
```suggestion
    You'll need to add the following "Redirect URI", making sure to update both your protocol to use either HTTP or HTTPS and your hostname as well. Keep in mind that this field is case sensitive.
```

Comment (pauloribeiro-codacy) [reply docs/configuration/git-providers/gitlab-enterprise.md:9]: The user must enter all the URIs, right? Are they entered one in each field, or all of them in the same field? :thinking:

File: docs/configuration/git-providers/gitlab-enterprise.md [hunk 3]
```diff
@@ -1,24 +1,34 @@
...
+    https://codacy.example.com/add/addService/GitLab
+    ```
 
-`https://codacy.example.com/add/addProvider/GitLabEnterprise`
+2.  Edit the file `values-production.yaml`, set `global.gitlab.enabled: "true"` and define the remaining values with the information obtained when you created the GitLab Application:
 
-`https://codacy.example.com/add/addService/GitLabEnterprise`
+    ```yaml
+    gitlab:
+      enabled: true
```
Comment (pauloribeiro-codacy) [inline docs/configuration/git-providers/gitlab-enterprise.md:22]: At least for GitHub Cloud this didn't work for me without the quotes!

```suggestion
      enabled: "true"
```

File: docs/configuration/git-providers/gitlab-enterprise.md (hunk 2 above)
Comment (jllopes) [reply docs/configuration/git-providers/gitlab-enterprise.md:9]: All of them are entered in the same field

**Pull request merged**